src/_posts/languages/nodejs/2000-01-01-start.md

```diff
@@ -356,6 +356,118 @@ To fix this issue, modify the `start` script by adding `-p $PORT`
 }
 ```
 
+### Next.js in Standalone Mode
+The Next.js built image can be quite large, easily exceeding 500MB. If you encounter issues with the image size, you can try "standalone" mode.
+
+Standalone mode can significantly reduce your image size. 
+* At the time of writing a fresh next.js build image is **547MB**,
+* A standalone image is **200MB**.
+
+#### Enable standalone mode:
+
+##### 1. Add the following to your `next.config.js` file:
+```js
+module.exports = {
+  output: 'standalone',
+}
+```
+or in `next.config.mjs`:
+```js
+const nextConfig = {
+    output: 'standalone',
+};
+```
+
+##### 2. Add `node_modules` to the `.slugignore` file
+
+This will prevent the `node_modules` directory from being included in the image.
+
+
+##### 3. Set start script in the package.json
+
+Start the server using node.
+
+```json
+{
+  "scripts": {
+    "start": "node .next/standalone/server.js"
+    // others scripts
+  }
+}
+```
+_Reason: after build node modules will have been removed, so the original command `next start` will error: "sh: 1: next: not found"_
```

Comment:
suggestion: IMO this should be a note block.
issue: folder name is not correctly spelled and should be between antiquotes
```suggestion
{% note %}
Reason: after the build `node_modules` will have been removed, so the original command `next start` will error: "sh: 1: next: not found"
{% endnote %}
```